### DISCUSSION TITLE
quick navigation to select folder to index

### DISCUSSION THREAD

**Discussion by loulou91** [photoprism/photoprism#4136] [Q&A]
Windows library-index allows the selection of the folder to index (windows 11 & Chrome).
It looks difficult to join any folder which is at the bottom of the list using scrolling.
Is there any tip to go directly and quickly to any place in the list of folders?

Thanks in advance.

**Comment by lastzero** [comment]
Selecting a folder to index is not really complicated and if you can type, you don't even need to scroll:

![Screenshot 2024-04-02 at 17 49 44](https://github.com/photoprism/photoprism/assets/301686/e0f91870-187c-48b4-8641-d8be99d01bab)

**Reply by loulou91** [reply]
I see your point!
It works fine if you perfectly know the path. If it is not the case and if you have hundred of folders it's not the same story.
In addition the displayed list of folder is flat (I mean no tree) so that scrolling may be very long.
